<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RamseyX Client</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="tbl20">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>110</y>
      <width>690</width>
      <height>325</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>690</width>
      <height>325</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>690</width>
      <height>325</height>
     </size>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>25</number>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
   </widget>
   <widget class="QTabWidget" name="tab20">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>91</y>
      <width>171</width>
      <height>30</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabTop20">
     <attribute name="title">
      <string>Top 20</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tabMe">
     <attribute name="title">
      <string>Me</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>261</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="labelStatus">
       <property name="text">
        <string>Status: &lt;span style=&quot; font-weight:600; color:#aa0000;&quot;&gt;Paused&lt;/span&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="txtTasksStatus"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>261</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_1">
       <property name="locale">
        <locale language="English" country="UnitedStates"/>
       </property>
       <property name="text">
        <string>What's Up:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="txtWhatsUp">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>90</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>90</height>
        </size>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Failed to fetch from server.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btnRefresh">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>1</y>
      <width>90</width>
      <height>28</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>90</width>
      <height>28</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>90</width>
      <height>28</height>
     </size>
    </property>
    <property name="text">
     <string>&amp;Refresh</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>450</y>
      <width>690</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/bmp/slogan&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>92</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Automatically&lt;br /&gt;update every&lt;br /&gt;ten minutes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnWhatsUp">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>28</height>
        </size>
       </property>
       <property name="text">
        <string>&amp;Update Now</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>150</y>
      <width>71</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QLabel" name="threadLabel">
       <property name="text">
        <string>1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QSlider" name="threadSlider">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBothSides</enum>
       </property>
       <property name="tickInterval">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Threads</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_5">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>692</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Project Overall:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="tblOverall">
       <property name="minimumSize">
        <size>
         <width>690</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>690</width>
         <height>50</height>
        </size>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="autoScroll">
        <bool>false</bool>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>25</number>
       </attribute>
       <attribute name="verticalHeaderHighlightSections">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>351</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnStart">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="locale">
        <locale language="English" country="UnitedStates"/>
       </property>
       <property name="text">
        <string>&amp;Start!</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAccount">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="locale">
        <locale language="English" country="UnitedStates"/>
       </property>
       <property name="text">
        <string>&amp;My Account</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>298</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QCheckBox" name="disableAutoUploadChk">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Disable auto upload</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="disableAutoDownloadChk">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Disable auto download</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="runOnStartupChk">
       <property name="text">
        <string>Run on system startup (recommended)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget_2</zorder>
   <zorder>layoutWidget_3</zorder>
   <zorder>layoutWidget_4</zorder>
   <zorder>layoutWidget_5</zorder>
   <zorder>layoutWidget_6</zorder>
   <zorder>layoutWidget_7</zorder>
   <zorder>tab20</zorder>
   <zorder>tbl20</zorder>
   <zorder>btnRefresh</zorder>
   <zorder>label_5</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="optionsMenu">
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="title">
     <string>&amp;Options</string>
    </property>
    <addaction name="resetBenchmarkAction"/>
    <addaction name="separator"/>
    <addaction name="quitAction"/>
   </widget>
   <widget class="QMenu" name="helpMenu">
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="mailingListAction"/>
    <addaction name="separator"/>
    <addaction name="aboutQtAction"/>
    <addaction name="aboutAction"/>
   </widget>
   <addaction name="optionsMenu"/>
   <addaction name="helpMenu"/>
  </widget>
  <action name="aboutAction">
   <property name="text">
    <string>About &amp;RamseyX Client...</string>
   </property>
  </action>
  <action name="resetBenchmarkAction">
   <property name="text">
    <string>&amp;Reset benchmark</string>
   </property>
  </action>
  <action name="mailingListAction">
   <property name="text">
    <string>&amp;Mailing list...</string>
   </property>
  </action>
  <action name="quitAction">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="aboutQtAction">
   <property name="text">
    <string>About &amp;Qt...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>btnStart</tabstop>
  <tabstop>btnAccount</tabstop>
  <tabstop>btnWhatsUp</tabstop>
  <tabstop>tab20</tabstop>
  <tabstop>txtWhatsUp</tabstop>
  <tabstop>txtTasksStatus</tabstop>
  <tabstop>tblOverall</tabstop>
  <tabstop>tbl20</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
